HOW_TO_CONTRIBUTE.md

```diff
@@ -34,10 +34,14 @@ Please open an issue or send an email to a maintainer before undertaking large s
 
 ### Commit Style
 
-Rabbot now uses [conventional commits](https://conventionalcommits.org/) so that going forward from 2.0.0, releases can be generated much more quickly (this includes automated generation of the CHANGELOG going forward).
+`foo-foo-mq` uses [conventional commits](https://conventionalcommits.org/), this is so releases can be generated much more quickly (this includes automated generation of the CHANGELOG going forward).
```

Comment:
```suggestion
`foo-foo-mq` uses [conventional commits](https://conventionalcommits.org/) so that releases can be generated much more quickly (this includes automated generation of the CHANGELOG going forward).
```